routes/users.js

```diff
@@ -1,9 +1,16 @@
-const express = require('express');
-const router = express.Router();
+const express = require('express')
+const router = express.Router()
+const mongoose = require('mongoose')
+const User = require('../models/user.js')
 
-/* GET users listing. */
-router.get('/', (request, response, next) => {
-  response.send('respond with a resource');
-});
+router.post( '/', ( request, response, next ) => {
+  User.create( request.body, ( error, user ) => {
```

Comment:
Missed this in previous commits; sorry `bout that - these should be promisified.  See harman and jesse, they've got an example in their PR